docset/windows/hyper-v/Resize-VHD.md

```diff
@@ -55,21 +54,21 @@ If the VHD belongs to the virtual SCSI chain, you **can** resize the virtual dis
 PS C:\> Resize-VHD -Path c:\BaseVHD.vhd -SizeBytes 1099511627776
 ```
 
-Expands the VHDX to 1 terabyte (assuming that the previous size was less than the new size prior to the command).
+Expands the VHD to 1 terabyte if the previous size was less than 1 terabyte. If it was larger, the cmdlet will report an error because it cannot shrink a VHD.
 
 ### Example 2
 ```
-PS C:\> Resize-VHD -Path c:\BaseVHDX.vhdx -SizeBytes 1099511627776
+PS C:\> Resize-VHD -Path c:\BaseVHDX.vhdx -SizeBytes 20GB
 ```
 
-Shrinks the VHDX to one terabyte (assuming that the virtual hard disk object associated with the file path has a **MinimumSize** less than or equal to 1TB).
+Changes the VHDX's size to 20 gigabytes (21,474,836,480 bytes). If it was larger, the cmdlet will only succeed if it had a **MinimumSize** less than or equal to 1TB 20 gigabytes.
```

Comment:
The text here says "equal to 1TB 20 gigabytes". Can you clarify?

Reply:
A typo, I created a change that dropped the 1TB reference.